plugins/outputs/wavefront/wavefront.go

```diff
@@ -123,20 +123,22 @@ func (w *Wavefront) Write(metrics []telegraf.Metric) error {
 			err := w.sender.SendMetric(point.Metric, point.Value, point.Timestamp, point.Source, point.Tags)
 			if err != nil {
 				if isRetryable(err) {
-					if flushErr := w.sender.Flush(); flushErr != nil {
-						w.Log.Errorf("wavefront flushing error: %v", flushErr)
+					// The internal buffer in the Wavefront SDK is full. To prevent data loss,
+					// we flush the buffer (which is a blocking operation) and try again.
```

Comment:
Can you help me understand this comment. Does a retryable error always mean that the buffer is full? Based on the comments on `isRetryable` everything is retryable except an empty metric name.

Reply:
Yeah, I believe `isRetryable` includes more than just the buffer full. From our testing, it is in that interval due where the buffer full error surfaces.